app/src/main/java/com/courseproject/tindar/ds/DatabaseHelper.java

```diff
@@ -677,8 +688,13 @@ public boolean checkLiked(String userId, String otherUserId) {
         return false;
     }
 
-    // precondition: userId < otherUserId. This is to avoid duplicates of
-    // (userId, otherUserId) and (otherUserId, userId)
+    /**
+     * Adds a matched pair to the database.
+     * Precondition: userId < otherUserId.
```

Comment:
I just realized it got excluded why we have this precondition. Can we include it back?

Reply:
done